back/app/models/custom_field.rb

```diff
@@ -261,7 +262,7 @@ def submittable?
 
   def printable?
     ignore_field_types = %w[page date files image_files point file_upload shapefile_upload topic_ids cosponsor_ids ranking matrix_linear_scale]
```

Comment:
Note: You'll need to remove `page` from here when you add to the field to the HTML template

Reply:
Just saying that if you pick up the ticket to add page details into the template, you'll need make sure that the list of printable fields includes page, but removing it from this line